tests/unit/sagemaker/remote_function/test_client.py

```diff
@@ -1498,7 +1499,6 @@ def test_consistency_between_remote_and_step_decorator():
     from sagemaker.workflow.function_step import step
 
     remote_args_to_ignore = [
-        "_remote",
```

Comment:
This is removed because we replace `remote.__code__.co_varnames` with `inspect.signature(remote).parameters.keys()`, and the latter retrieves the function parameters precisely?

Reply:
Yes, right. The change was also needed since now that we are decorating `remote()` decorator with `_telemetry_emitter()` decorator, the `func.__code__.co_varnames` doesn't seem to return correct list of args. So I fixed the unit-test.